templates/superset_config.py

```diff
@@ -99,15 +99,17 @@ class CeleryConfig(object):
 CELERY_CONFIG = CeleryConfig
 
 FEATURE_FLAGS = {
-    'ALERTS_ATTACH_REPORTS': True,
-    'DASHBOARD_CROSS_FILTERS': True,
-    'DASHBOARD_RBAC': True,
-    'EMBEDDABLE_CHARTS': True,
-    'SCHEDULED_QUERIES': True,
-    'ESTIMATE_QUERY_COST': True,
-    'ENABLE_TEMPLATE_PROCESSING': True,
-    'ALERT_REPORTS': True,
-
+    flag_name: bool(os.getenv(flag_name, '').lower())
```

Comment:
The value of the flag will be True as long as a env variable is set. Just checking if this is the wanted behavior?

Also, if this is the required behavior, you could make it clear with:

```python
os.getenv(flag_name) is not None for ...
```

Reply:
Thank you, I actually want it to have a value of False if the string is `false`, I've corrected this now.